package.json

```diff
@@ -36,24 +36,28 @@
   },
   "homepage": "https://github.com/mypurecloud/webrtc-stats-gatherer#readme",
   "devDependencies": {
-    "@types/jest": "^26.0.14",
-    "jest": "^26.4.2",
-    "pre-commit": "^1.1.3",
-    "pre-push": "^0.1.1",
-    "rimraf": "^3.0.2",
-    "run-sequence": "^2.0.0",
-    "ts-jest": "^26.4.1",
-    "tslint": "^6.1.3",
-    "tslint-config-semistandard": "^8.0.1",
-    "typescript": "^4.0.3",
-    "webpack": "^4.41.2",
-    "webpack-cli": "^3.1.0"
+    "@eslint/js": "^9.2.0",
+    "@types/jest": "^29.5.12",
+    "eslint": "^8.57.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "npm-check": "^6.0.1",
+    "pre-commit": "^1.2.2",
+    "pre-push": "^0.1.4",
+    "prettier": "3.2.5",
```

Comment:
Prettier suggests pinning to an exact version since even a patch update can change formatting.